<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstRunWizard</class>
 <widget class="QWizard" name="FirstRunWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ClamUI - Einrichtungsassistent</string>
  </property>
  <property name="windowIcon">
   <iconset theme="tools-wizard"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::HaveHelpButton|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>2. Schritt: Das Programm &lt;b&gt;FreshClam&lt;/b&gt; einrichten.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>1. Schritt: Den Virenscanner &lt;b&gt;ClamAV&lt;/b&gt; einrichten.</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>3. Schritt: Ihre Angaben überprüfen und die Konfiguration abschließen.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Ausführungsoptionen</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_HideWindow">
           <property name="toolTip">
            <string>Das Programm läuft im Systemabschnitt der Kontrollleiste und es wird nach dem starten kein Fenster angezeigt. </string>
           </property>
           <property name="text">
            <string>Fenster beim Start nicht anzeigen.</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_Autostart">
           <property name="toolTip">
            <string>Startet ClamUI automatisch nach dem einloggen in KDE.</string>
           </property>
           <property name="text">
            <string>ClamUI automatisch starten.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_ExternalDevice">
           <property name="toolTip">
            <string>Externe Datenträger bei Anschluss automatisch prüfen.</string>
           </property>
           <property name="text">
            <string>Externe Datenträger bei Anschluss automatisch prüfen.</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox_IconOnDesktop">
           <property name="toolTip">
            <string>Erstellt auf dem Desktop eine Verknüpfung zum Programm.</string>
           </property>
           <property name="text">
            <string>Icon auf dem Desktop anzeigen.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>ClamAV Prüfungsinterval</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,1">
            <item row="0" column="1">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Viren&amp;suche festlegen:</string>
              </property>
              <property name="buddy">
               <cstring>comboBox_Scheduling</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="comboBox_Scheduling"/>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QFrame" name="frame_DateTime">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QGridLayout" name="gridLayout_DateTime" columnstretch="0,0,0">
                 <item row="1" column="1">
                  <widget class="QTimeEdit" name="timeEdit_ScheduleTime">
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="currentSectionIndex">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDateEdit" name="dateEdit_ScheduleDate">
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="displayFormat">
                    <string>dd.MM.yyyy</string>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_ScheduleDate">
                   <property name="text">
                    <string>Datu&amp;m:</string>
                   </property>
                   <property name="buddy">
                    <cstring>dateEdit_ScheduleDate</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_ScheduleTime">
                   <property name="text">
                    <string>Uhr&amp;zeit:</string>
                   </property>
                   <property name="buddy">
                    <cstring>timeEdit_ScheduleTime</cstring>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Automatisches aktualisieren der Virendatenbank</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="2" colspan="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Freshclam Status</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBox_FreshClam">
           <property name="toolTip">
            <string>Startet das Programm freshclam im Hintergrund und überprüft in regelmäßigen Abständen die Aktualität der Virendefinitionen. </string>
           </property>
           <property name="text">
            <string>freshclam im Hintergrund starten.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Wie häufig soll nach einer aktuellen Virendatenbank gesucht werden?</string>
           </property>
           <property name="text">
            <string>Wie oft am Tag aktuallisieren:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_FreshClamUpdate">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Wie häufig soll nach einer aktuellen Virendatenbank gesucht werden?</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>48</number>
           </property>
           <property name="value">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="pushButton_FreshclamStatus">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_FreshclamStatus">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_2"/>
 </widget>
 <resources/>
 <connections/>
</ui>
